astropy/io/fits/tests/test_core.py

```diff
@@ -51,10 +51,7 @@ def test_byteswap(self):
         p = fits.PrimaryHDU()
         lst = fits.HDUList()
 
-        n = np.zeros(3, dtype='i2')
-        n[0] = 1
-        n[1] = 60000
-        n[2] = 2
+        n = np.array([1, 60000, 0]).astype('i2')
```

Comment:
I think it has been pure luck that this test worked -- that it wrapped around automatically. I think we should just be explicit and write `n[1] = 60000 - 65536`

Reply:
OK, that's true. Maybe use your more explicit route?